src/PSPackageProject.psm1

```diff
@@ -93,6 +93,102 @@ function Invoke-PSPackageProjectTest
     )
 
     ## TODO implement calling tests
+    Invoke-BinSkim
+}
+
+function Invoke-BinSkim
```

Comment:
Do these functions need to be exported in the manifest?

Reply:
I think, no. `-Type StaticAnalysis` should be the "public" interface.

Reply:
No